nextflow-modules/modules/seqtk/main.nf

```diff
@@ -0,0 +1,54 @@
+process seqtk_sample {
+  tag "${sampleID}"
+  scratch params.scratch
+
+  input:
+    tuple val(sampleID), path(reads), val(sample_size)
```

Comment:
See comment about separating channels

Reply:
Yes but this was the best solution that causes the least amount of issues

Reply:
Nextflow requires input channels to be defined. The `sample_size` parameter can be either an int, a float or a null which causes nextflow to crash if set as a separate channel but its allowed if passed with the rest of the data.

Reply:
Quast has a channel that can be null (`referenceGenome`) when using the `streptococcus` profile, can't we copy the same logic?